FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/FreeRTOS_IP_Private.h

```diff
@@ -401,6 +401,7 @@ socket events. */
 		(right) = tmp; \
 	} while ( ipFALSE_BOOL )
 
+/* WARNING: Do NOT use this macro when the array was received as a parameter. */
```

Comment:
Hmmm... Posting an opinion. 

I'd probably define our APIs to accept both array size and pointer. And probably remove this macro, since not so many places in our library code uses this macro. (Two occurrences in ```FreeRTOS_TCP_IP.c```, which we would care about. And a couple in porting layer which, which is not our top concern.) 

Though here, I'd happily accept this comment. :)